yt/frontends/adaptahop/io.py

```diff
@@ -50,7 +50,7 @@ def _read_particle_coords(self, chunks, ptf):
             if pcount == 0:
                 continue
             pos = self._get_particle_positions()
-            yield ptype, [pos[:, i] for i in range(3)]
+            yield ptype, [pos[:, i] for i in range(3)], 0.0
```

Comment:
This will break any third party particle frontend, I think. I don't know that there are any, and in any case it is trivial to mitigate but it needs to be mentioned in our next release notes, I think.

Reply:
Agreed!

Reply:
going to add it to a draft release note for yt 4.1